contents/docs/self-host/index.mdx

```diff
@@ -8,11 +8,64 @@ showTitle: true
+
+Because we don't manage self-hosted instances or provide paid support plans for them, [we don't offer any sort of guarantees](/docs/self-host/open-source/disclaimer) around it working in certain ways on your infrastructure, and you assume all responsibility and risk for your use of the product and the stack.
+
+## Should I use self-hosted PostHog or PostHog Cloud?
+
+Good question! We've found that PostHog Cloud is far and away the best experience for the vast majority of our users. However, some people still want to self-host, and we're here for that! Here's a quick flow chart to help you understand if self-hosting PostHog is for you:
+
+```mermaid
+---
+config:
+  layout: elk
+---
+flowchart TD
+    A@{ label: "I'm a die-hard self-hoster" } --> B("Do you expect to ingest more than 300k events, 1k recordings, 300k /decide API calls into the instance?")
+    n6["What makes you interested in self-hosting PostHog?"] --> A & n7["I have data privacy requirements / desires"] & n8@{ label: "I'm worried about cost" }
+    n8 --> n9["Will you have more than 1M events, 5k recordings etc per month?"]
+    n9 -- No --> n10["PostHog Cloud is free for 1M events, 5k recordings, 250 surveys, and more every single month. See posthog.com/pricing for more."]
+    n10 --> n11@{ label: "You'll probably be better off with PostHog Cloud" }
+    n7 --> n12["PostHog Cloud has both EU and US regions for GDPR and BAAs available for HIPPA-compliant companies."]
+    n12 -- Cool --> n11
+    B -- Yes --> n13["Is managing infrastructure one of your core competencies?"]
+    B -- No --> n3@{ label: "Are you okay with accepting the risk for potential data loss?&nbsp;<span style=\"font-family:\">(we have no guarantees for self-hosted instances)</span>" }
+    n13 -- No --> n11
+    n9 -- Yes --> n14@{ label: "Have you compared the pricing for your anticipated volume vs the cost of a devops engineer's time?" }
+    n14 -- "Yes, and it's cheaper to do this in-house" --> n15@{ label: "We've literally never seen this math work out. However..." }
+    n15 --> n13
+    n12 -- It's something else --> n16["We host all lots of companies with privacy needs, feel free to send a message with your requirements and we can go from there."]
+    n16 --> n11
+    n14 -- No --> n18["Use the calculator on posthog.com/pricing, then come back"]
+    n18 -- "Going back to this question..." --> n14
+    n14 -- Yes, and it's cheaper to use Posthog Cloud --> n19["Because we spread the cost of managing infra across all our customers, PostHog Cloud is usually far cheaper than doing it yourself!"]
+    n19 --> n11
+    n13 --> n3
+    n3 --> n20@{ label: "We don't offer customer support for product, infrastructure, or other questions for self-hosted instances. That alright with you?" }
+    n20 -- No --> n11
+    n20 --> n21["Are you cool with just using the features that are available on the free plan? All paid-plan features are Cloud-only. But of course, you do have access to the source code ;)"]
```

Comment:
```suggestion
    n20 --> n21["Are you cool with just using the features that are available on the free plan? All paid-plan features are Cloud-only."]
```

Reply:
ha I kinda liked acknowledging that people think they can or will do this